Installation/install-in-a-virtual-machine/advanced-qemu-config.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: 'Advanced tweaks for qemu'
+date: '2021-10-27'
+---
+# Advanced configuration for qemu
+
+Qemu is a very powerfull tool that may be confusing for some people. however when using it we can get some very detailed setups, and advanced setups, this page will document some more advanced features that folk may wish to utilize with qemu. the majority of this page will assume running on linux. many of these will not be compatible with windows. 
```

Comment:
QEMU is a very powerful tool that may be confusing for some users. When using it we can get some very detailed and advanced setups. This page will document some more advanced features that users may wish to utilize with QEMU. The majority of this page will assume the user is running on Linux as many of these options may not be compatible on Windows.